packages/meditrak-app/app/assessment/selectors.js

```diff
@@ -127,10 +127,12 @@ export const getArithmeticResult = (state, arithmeticQuestionId) => {
     if (valueTranslation[questionId] && valueTranslation[questionId][answer] !== undefined) {
       return valueTranslation[questionId][answer]; // return translated answer if there's any
     }
-    if (answer !== undefined) {
-      return answer; // return raw answer
+    // return raw answer if it's a number, else 0 (e.g. if no valueTranslation provided for the question and this specific answer when answer is non-numeric)
+    if (answer !== undefined && answer !== null) {
+      return isNaN(answer) ? 0 : answer; // return raw answer
```

Comment:
Do we protect from empty string somewhere?

Reply:
Such a javascript issue to have 🙃 I have added a fix for that, thanks!

Reply:
Cheers, looks good 👍